msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -20,6 +20,16 @@ Your bot receives messages from Teams using the `Text` property and it sends sin
 
 For more information, see [user attribution for bot messages](/microsoftteams/platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit?tabs=dotnet%2Cdotnet-1#user-attribution-for-bots-messages).
 
+The following table lists the events that your bot can receive and take action on:
+
+| Event type | Payload object | Scope |
+| ---- | ---------| ------ |
+| [Receive a message](#receive-a-message) | `Activity` | All |
```

Comment:
Can we change the table to:

Event Type | Payload Object | Scope
Receive a message activity | Message activity | All
Receive edit message activity | Message edit activity | All
Receive undelete message activity | Message undelete activity | All
Get soft delete message activity | Message soft delete activity | All

Reply:
updated as suggested